### ISSUE TITLE
Add Apply Lambda

### ISSUE THREAD

**Issue by carlosjpc** [ivansabik/panditas#23]
Not sure it's possible, but would greatly expand flexibility

**Comment by ivansabik** [comment]
Yeah my only concern would be running unsafe code but since we would be running in docker not sure how much hard that could do. Also people abusing of lambdas but I can see those as being really helpful
